curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

```diff
@@ -89,7 +89,15 @@ assert.lengthOf(document.querySelectorAll('h1'), 1);
 You should have a `p` element below your `h1` element with text describing your recipe.
```

Comment:
```suggestion
You should have a `p` element below your `h1` element.
```

Reply:
Why did you not accept this @CracktheDom ?